benchmark/babel.js

```diff
@@ -0,0 +1,30 @@
+import path from 'path'
+import Benchmark from 'benchmark'
+import {transformFile} from 'babel-core'
+
+import plugin from '../src/babel'
+
+const transform = (file, opts = {}) => (
+  new Promise((resolve, reject) => {
+    transformFile(path.resolve(__dirname, file), {
```

Comment:
Yeah but I am not sure if `benchmark` supports async setup https://github.com/bestiejs/benchmark.js/pull/174

I can read the file synchronously before exporting the benchmark wdyt?